tasks/eth/fp-recovery/004-recover-bonds/input-template.json

```diff
@@ -0,0 +1,33 @@
+{
+  "chainId": 1,
+  "metadata": {
+    "name": "Recover all funds in the DelayedWETH contract",
```

Comment:
NIT: this is not recovering **ALL** the funds, but the **desired amount** passed in the input.

Reply:
Re-worded these descriptions